### PR TITLE
Update exporting.md

### PR DESCRIPTION
**Pull request by trolologuy**
Added some precisions on how to access the environment variable's values and why it would probably be a good idea to backup them as well.

### CODE REVIEW

File: docs/src/tutorials/exporting.md [hunk 1]
```diff
@@ -59,3 +59,27 @@ See the [`rsync` documentation](https://download.samba.org/pub/rsync/rsync.html)
 The mechanism for downloading from each service (such as your database) varies.  For services designed to hold non-persistent information (such as Redis or Solr) it's generally not necessary to download data as it can be rebuilt from the primary data store.
 
 To download data from persistent services ([MySQL](/configuration/services/mysql.md), [PostgreSQL](/configuration/services/postgresql.md), [MongoDB](/configuration/services/mongodb.md), or [InfluxDB](/configuration/services/influxdb.md)), see each service's page for instructions.
+
+
+## Get the environment variables
+
+If your project uses some environment variable (tokens, ...) it can be helpful to backup them if you didn't store them separately.
+
+As precised in the management console, several possibilities exist for the environment variables.
```
Comment (Crell) [inline docs/src/tutorials/exporting.md:68]:
```suggestion
As stated in the management console, several possibilities exist for the environment variables.
```

I think that's what you mean here?

File: docs/src/tutorials/exporting.md [hunk 2]
```diff
@@ -59,3 +59,27 @@ See the [`rsync` documentation](https://download.samba.org/pub/rsync/rsync.html)
...
+> • Variables beginning with env: will be exposed as Unix environment variables
+
+> • Variables beginning with php: will be interpreted as php.ini directives.
+
+> • All other variables will be part of the environment $PLATFORM_VARIABLES variable
+More details can be found [here](https://docs.platform.sh/administration/web/configure-environment.html#variables
+)
```
Comment (Crell) [inline docs/src/tutorials/exporting.md:75]: These should be a normal bulleted list, not a quoted pseudo-list.

File: docs/src/tutorials/exporting.md (hunk 1 above)
Comment (trolologuy) [reply docs/src/tutorials/exporting.md:68]: yes, my bad. "Stated" is much better.

**Comment by trolologuy** [comment]
Should be clearer now.

File: docs/src/tutorials/exporting.md [hunk 3]
```diff
@@ -59,3 +59,25 @@ See the [`rsync` documentation](https://download.samba.org/pub/rsync/rsync.html)
 The mechanism for downloading from each service (such as your database) varies.  For services designed to hold non-persistent information (such as Redis or Solr) it's generally not necessary to download data as it can be rebuilt from the primary data store.
 
 To download data from persistent services ([MySQL](/configuration/services/mysql.md), [PostgreSQL](/configuration/services/postgresql.md), [MongoDB](/configuration/services/mongodb.md), or [InfluxDB](/configuration/services/influxdb.md)), see each service's page for instructions.
+
+
```
Comment (Crell) [inline docs/src/tutorials/exporting.md:63]:
```suggestion

```

Single line between paragraphs, not multiple.

File: docs/src/tutorials/exporting.md [hunk 4]
```diff
@@ -59,3 +59,25 @@ See the [`rsync` documentation](https://download.samba.org/pub/rsync/rsync.html)
...
+## Get the environment variables
+
+If your project uses some environment variable (tokens, ...) it can be helpful to backup them if you didn't store them separately.
+
+As stated in the management console, several possibilities exist for the environment variables.
+• Variables beginning with `env`: will be exposed as Unix environment variables
+• Variables beginning with `php`: will be interpreted as php.ini directives.
+• All other variables will be part of the environment $PLATFORM_VARIABLES variable
+More details can be found [here](https://docs.platform.sh/administration/web/configure-environment.html#variables
+)
```
Comment (Crell) [inline docs/src/tutorials/exporting.md:73]:
```suggestion
As stated in the management console, several possibilities exist for the environment variables.

* Variables beginning with `env:` will be exposed as Unix environment variables
* Variables beginning with `php:` will be interpreted as `php.ini` directives.
* All other variables will be part of the environment `PLATFORM_VARIABLES` variable

More details can be found [on the environment page](https://docs.platform.sh/administration/web/configure-environment.html#variables)
```

File: docs/src/tutorials/exporting.md [hunk 5]
```diff
@@ -59,3 +59,24 @@ See the [`rsync` documentation](https://download.samba.org/pub/rsync/rsync.html)
 The mechanism for downloading from each service (such as your database) varies.  For services designed to hold non-persistent information (such as Redis or Solr) it's generally not necessary to download data as it can be rebuilt from the primary data store.
 
 To download data from persistent services ([MySQL](/configuration/services/mysql.md), [PostgreSQL](/configuration/services/postgresql.md), [MongoDB](/configuration/services/mongodb.md), or [InfluxDB](/configuration/services/influxdb.md)), see each service's page for instructions.
+## Get the environment variables
```
Comment (Crell) [inline docs/src/tutorials/exporting.md:62]:
```suggestion

## Get the environment variables
```

**Review by Crell — APPROVED** [review]

**Pull request merged**